lib/multihash.ex

```diff
@@ -41,7 +43,8 @@ defmodule Multihash do
   # Error strings
   @error_invalid_digest_hash "Invalid digest or hash"
   @error_invalid_multihash "Invalid multihash"
-  @error_invalid_length "Invalid length of provided hash function"
+  @error_invalid_length "Invalid length"
+  @error_invalid_trunc_length "Invalid truncation length is longer than digest"
```

Comment:
The error doesn't sound right. I think it should be
`Invalid truncation length` as it is being returned if `l <=0 or l >= default_length`